plugins/uefi-capsule/efi/generate_binary.py

```diff
@@ -72,6 +128,57 @@ def _run_genpeimg(args):
         default="x86_64",
         help="EFI architecture",
     )
+    parser.add_argument(
+        "--project-name",
+        help="SBAT project name",
+    )
+    parser.add_argument(
+        "--project-version",
+        help="SBAT project version",
+    )
+    parser.add_argument(
+        "--sbat-version",
+        default=0,
```

Comment:
whatever the conclusion is for default to 0 or 1 index this might need updating.